src/app/+display/display.component.ts

```diff
@@ -133,6 +133,10 @@ export class DisplayComponent implements OnDestroy, OnInit {
             })
           );
 
+          this.discoveryView.subscribe((val) => {console.log('discoveryView:', val)})
+          this.individual.subscribe((val) => {console.log('individual:', val)})
+
```

Comment:
Would we need these log statements for this feature.

Reply:
These debugging logs should be removed.

Reply:
Logging statements removed, please confirm.